doc/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Delete this sentence after you read the "Issues" paragraph from the
+"guidelines for contributing" link above.
+
+If you are reporting a bug, please try to fill in the following, otherwise
+remove it along with this sentence.
+
+| Question       | Answer
+|----------------|-------------------------------------------------------------
+| Bundle version | composer show --installed | grep sonata-project/admin-bundle
```

Comment:
`--installed` is deprecated since composer 1.0. This is now the default behaviour.